src/static_canvas.class.js

```diff
@@ -232,15 +232,15 @@
      * @private
      */
     _isRetinaScaling: function() {
-      return (fabric.devicePixelRatio !== 1 && this.enableRetinaScaling);
+      return (fabric.devicePixelRatio > 1 && this.enableRetinaScaling);
     },
 
     /**
      * @private
      * @return {Number} retinaScaling if applied, otherwise 1;
      */
     getRetinaScaling: function() {
-      return this._isRetinaScaling() ? fabric.devicePixelRatio : 1;
+      return Math.max(1, fabric.devicePixelRatio);
```

Comment:
ok this was the error.
We lessen the importance of the boolean here.
If retina scaling is set as false, whatever is the devicePixelRatio, we return 1.
My bad for not noticing it.

Reply:
Probably we can fix it like this?
```js
this._isRetinaScaling() ? Math.max(1, fabric.devicePixelRatio) : 1
```